addons/addon-base-raas/packages/base-raas-services/lib/schema/create-egress-store.json

```diff
@@ -0,0 +1,75 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
```

Comment:
apply validation check for all the properties. This will most likely be flagged by AppSec. There are examples all across SWB code where this is being done.

Reply:
Updated in commit: 752261efcabd292814398a679f3c83f7ceffa328